src/nni_manager/training_service/pai_base/pai/paiTrainingService.ts

```diff
@@ -0,0 +1,326 @@
+/**
```

Comment:
why this `pai` dir is under `pai_base` dir?

Reply:
`pai` and `paiYarn` are two trainingService, they inherit from `paiBase`